### PR TITLE
AkiOhtani's clusters and sunburst chart

### PR DESCRIPTION
**Pull request by AkiOhtani**
I added sunburst.html,  enrgy.json and energy_consumption_vis.ipynb.
I need more data processing in order to complete sunburst .

### CODE REVIEW

**Comment by shirakia** [comment]
Thank you for your PR!!

Using d3 is a good idea but adding d3 itself to the repo is a bit heavy. Can you exclude it and use CDN?

**Comment by AkiOhtani** [comment]
Thank you so much for your advise.
I removed d3 and revised link address!

**Comment by shirakia** [comment]
To remove it from git repo, you should do `git rm -r d3`. Could you try it?

**Comment by AkiOhtani** [comment]
OK. I did.

**Comment by AkiOhtani** [comment]
Could you check it?

**Comment by shirakia** [comment]
Thanks. It's fine. Now we're gonna see the PR.

**Comment by AkiOhtani** [comment]
Thank you!

File: challenge2/analysis/AkiOhtani/energy.json [hunk 1]
```diff
@@ -0,0 +1,1235 @@
...
+                        ], 
+                        "name": "Soury"
+                    }
+                ], 
+                "name": "Saturday"
+            }
+        ], 
+        "name": "large"
+    }
+]
```
Comment (thedimlebowski) [inline challenge2/analysis/AkiOhtani/energy.json:2138]: This file is very obscure to me. I expect more energy-related stuff and less "Roasted Peanuts" content.

File: challenge2/analysis/AkiOhtani/index.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  };
+}
+
+function updateArc(d) {
+  return {depth: d.depth, x: d.x, dx: d.dx};
+}
+
+d3.select(self.frameElement).style("height", margin.top + margin.bottom + "px");
+
+</script>
```
Comment (thedimlebowski) [inline challenge2/analysis/AkiOhtani/index.html:171]: This looks inspired by http://bl.ocks.org/mbostock/5944371

**Comment by AkiOhtani** [comment]
I referred to this page: http://www.jasondavies.com/coffee-wheel/.

**Comment by AkiOhtani** [comment]
Sorry, energy.json file is incomplete yet.

**Comment by thedimlebowski** [comment]
Dear AkiOhtani,

Thank you very much for addressing the challenge.

General comments:
- You explored many ways of plotting the dataset which is interesting.
- Your graphs were of acceptable quality even if the representation choice was not very standard.
- Your notebook is very messy, with error messages, meaningless output such as:
  image
- There is not a single line of interpretation of your results.

Thank you again for addressing the challenge. We'll be in touch shortly.

Cheers,
Dimitry - CEDL Data-Scientist.

**Pull request merged**